exercises/practice/square-root/.meta/example.ua

```diff
@@ -0,0 +1 @@
+SquareRoot ← |1 -1⊙(◌◌)⍢(: ⁿ2. +1 ⊙◌|≥⋅⊙⊙) 1 1
```

Comment:
I expected a classic Newton implementation, but this is trying every positive number until you hit the correct one, right?

 guess the extra work is not necessary as the tests only focus on integers, so: good work, or in German: [ein gutes Pferd springt nicht höher, als es muss](https://en.wiktionary.org/wiki/ein_gutes_Pferd_springt_nicht_h%C3%B6her,_als_es_muss)

Reply:
Nice German proverb! But yes, this is a classic integer only algorithm